router/customdestinationmanager/customdestinationmanager.go

```diff
@@ -136,7 +136,7 @@ func (customManager *CustomManagerT) send(jsonData json.RawMessage, client inter
 
 		switch {
 		case kvManager.ShouldSendDataAsJSON(config):
-			_, err = kvManager.SendDataAsJSON(jsonData)
+			_, err = kvManager.SendDataAsJSON(jsonData, config)
```

Comment:
I don't see `useJSONModule` being set in the redis creation flow. Do we actually prompt the user for this option?

Reply:
Yes. But this is behind a feature-flag may be could be the reason for not seeing it in redis creation flow.

Reply:
Resolving the conversation